subsys/bluetooth/host/iso.c

```diff
@@ -759,6 +762,9 @@ static struct net_buf *iso_data_pull(struct bt_conn *conn,
 		LOG_DBG("channel has been disconnected");
 		__ASSERT_NO_MSG(b == frag);
 
+		/* Service other connections */
+		bt_tx_irq_raise();
```

Comment:
Do we need this in the `if (bt_buf_has_view(frag)) {` clause? 

It says it shouldn't happen, but wouldn't the same issue occur if that happens?

Reply:
I haven't managed to exercise it yet, so I'm not sure.